addons/core/translations/errors/en-us.yaml

```diff
@@ -17,3 +17,5 @@ generic:
   title: Error
 authentication-failed:
   title: Authentication Failed
+origin-verification-failed:
+  description: Could not validate the provided origin URL as Boundary URL. Please check origin URL and try again.
```

Comment:
Unable to connect to the specified origin.  Please check the URL and try again.

Reply:
Updated.